CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## HEAD
+
+- **Fix** Fix server's port argument.
```

Comment:
Let's be more clear that what's fixed is the `-p, --port` argument for the CLI. (Since the `port` option in `batfish.config.js` already worked just fine.)